tardis/model/geometry/tests/test_radial1d.py

```diff
@@ -0,0 +1,79 @@
+from astropy import units as u
+import numpy as np
+
+from tardis.model.geometry.radial1d import HomologousRadial1DGeometry
+
+import pytest
+
+
+@pytest.fixture(scope="function")
+def homologous_radial1d_geometry():
+    velocity = np.arange(8000, 21000, 1000) * u.km / u.s
+    v_inner = velocity[:-1]
+    v_outer = velocity[1:]
+    time_explosion = 5 * u.day
+    geometry = HomologousRadial1DGeometry(
+        v_inner, v_outer, v_inner[0], v_outer[-1], time_explosion
+    )
+    return geometry
+
+
+def test_v_indices(homologous_radial1d_geometry):
+    # Testing if the indices returned are correct when inner and outer
+    # boundary are on the innermost and outermost shell
+
+    homologous_radial1d_geometry.v_inner_boundary = (
+        homologous_radial1d_geometry.v_inner[0]
+    )
+    homologous_radial1d_geometry.v_outer_boundary = (
+        homologous_radial1d_geometry.v_outer[-1]
+    )
+    assert homologous_radial1d_geometry.v_inner_boundary_index == 0
+    assert homologous_radial1d_geometry.v_outer_boundary_index == 12
```

Comment:
I think these values should be parametrized in some way, or made more clear where they come from

Reply:
this one says outdated for me.